packages/flame/lib/src/devtools/connectors/overlay_nav_connector.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:convert';
+import 'dart:developer';
+
+import 'package:flame/src/devtools/dev_tools_connector.dart';
+
+/// The [OverlayNavConnector] is responsible of getting the names of all
+/// registered overlays and navigating to the overlay with the given name.
+class OverlayNavConnector extends DevToolsConnector {
```

Comment:
We're trying not use and abbreviations in this repository, so write out the whole word here.